src/term-spec.js

```diff
@@ -32,13 +32,26 @@ declare export class NamedObjectProperty extends ObjectProperty {
   name: PropertyName;
 }
 declare export class MethodDefinition extends NamedObjectProperty {
+  type?: TypeNode;
   body: FunctionBody;
 }
 
+declare export class AbstractMethodDefinition extends Term {
```

Comment:
Should this be `TsAbstractMethodDefinition`?

Reply:
Yes, that makes a lot of sense. The change is already in.